pkg/kv/bulk/sst_batcher.go

```diff
@@ -345,6 +345,25 @@ func (b *SSTBatcher) AddMVCCKeyWithImportEpoch(
 	return b.AddMVCCKey(ctx, key, b.valueScratch)
 }
 
+func (b *SSTBatcher) AddMVCCKeyLDR(
```

Comment:
i can add a little unit test in sst_batchet_test.go, if folks like this approach.